Atomic/containers.py

```diff
@@ -73,6 +73,8 @@ def cli(subparser):
     updatep.set_defaults(_class=Containers, func='update')
     updatep.add_argument("container",
                          help=_("Specify one or more containers. Must be final arguments."))
+    updatep.add_argument("--to", dest="to", default=None,
+                         help=_("Update to a different image (useful to upgrade to a different tag)"))
```

Comment:
I think --tag is not complete, as we can update to another image in a different registry, like:
```
atomic install --name etcd gscrivano/etcd
atomic containers update etcd --to rhel/etcd
```

Should we call it `--rebase`?